transport/processing/processing_component.py

```diff
@@ -1,9 +1,10 @@
+from dataclasses import dataclass, field
```

Comment:
yet another stylistic remark: I personally prefer to stick with `import a.b.c` import-style, and use import-aliases if import-names would become too long (pbly this is not a generally-approved practice in python-community - so from-import is in general also fine w/ me)

however, we do typically sort our imports alphabetically (not sure, however, how this fits with mixing import and from-import..) :-/